proposal/model-registry.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Model Registry
+---
+
+**Introduction**
+
+Model Registry in an ML DevOps environment is a central repository for storing metadata about ML models.
+
+During the development and deployment life cycle of an ML model, various artifacts and metadata are created or associated with the ML model.
+For example, a *Data Scientist* may create training data, parameters, metrics during development. During deployment, an *MLOps Engineer* may work with artifacts such as versioned model data. Model serving will generate metadata about hyper parameters and metrics from inference. 
+Execution of actions and events performed during the model's lifecycle, for example in a pipeline that executes various steps to process and transform data, can also be stored as metadata. 
+
+There needs to be a central repository for all such metadata so that it can provide visibility across the entire lifecycle of the model. 
+It should be able to provide lineage and provenance for any artifact or metadata related to an ML model. 
+
+**Design Goals**
+
+Model Registry should be a passive repository which receives and stores ML model related metadata pushed to the registry.
+The registry may have simple rules for validating metadata, but it does not take any actions or perform any orchestration based on this metadata. 
+The metadata can be pushed to the repository using client libraries in code written by ML users or automatically generated by other ODH components in the platform. For example, metadata about the deployment status of an ML Model. 
+
+**User Use Cases**
+- (Optional) *Data Scientists* need self-service access to a Model Registry to store Experiments and Runs for Tracking. 
```

Comment:
remove (optional)

*Data Scientists* need access to a Model Registry to store information about Experiments and Runs for Tracking and reproducibility.

Reply:
Fixed.